go/libraries/doltcore/table/untyped/xlsx/reader.go

```diff
@@ -51,18 +51,29 @@ func OpenXLSXReader(nbf *types.NomsBinFormat, path string, fs filesys.ReadableFS
 
 	data, err := getXlsxRows(path, info.SheetName)
 	if err != nil {
-		return nil, err
+		return nil, false, err
 	}
 
-	_, sch := untyped.NewUntypedSchema(colStrs...)
-
-	decodedRows, err := decodeXLSXRows(nbf, data, sch)
-	if err != nil {
-		r.Close()
-		return nil, err
+	fileMatchesSchema := true
+	var decodedRows []row.Row
+	var inSch schema.Schema
+	if outSch != nil {
+		inSch = outSch
+		decodedRows, err = decodeXLSXRows(nbf, data, outSch)
+		if err != nil {
+			fileMatchesSchema = false
```

Comment:
I didn't want to return the error here because when you're updating a table (as opposed to replacing) you don't get an error if the schema of the new file doesn't match the original table schema, it just doesn't update any rows. I was using similar logic [here for JSON files](https://github.com/liquidata-inc/dolt/pull/86/files/10aacb127386e2bb16255b03c187f37002e99378#diff-a88f89d9abb3e934ef37618a5bb46466R82), but let me know if you want me to change that

Reply:
Ended up changing this function back to how it was before, so this is irrelevant now